numba/pycc/modulemixin.c

```diff
@@ -13,6 +13,8 @@
 #define NUMBA_EXPORT_FUNC(_rettype) VISIBILITY_HIDDEN _rettype
 #define NUMBA_EXPORT_DATA(_vartype) VISIBILITY_HIDDEN _vartype
 
+#define NUMBA_COMPILING_AOT
```

Comment:
I think elsewhere the `PYCC` prefix is in use, would `PYCC_COMPILING` fit better?